docs/kernel.rst

```diff
@@ -13,8 +13,10 @@ The following steps should be performed for all of the `recommended hardware`_:
 
 #. Install the new kernel packages on your *Monitor Server*, then reboot. Verify with ``uname -r`` that you are using the new kernel.
 #. If it doesn't boot, see the `Troubleshooting Kernel Updates`_ documentation.
-#. Verify ``paxtest`` doesn't return any errors nor warnings.
-#. Verify the `spectre-meltdown-checker`_ doesn't return any errors nor warnings.
+#. Install the ``paxtest`` package, run with ``sudo paxtest blackhat`` and verify it doesn't
```

Comment:
Just a small nitpick, but [our guidelines indicate that we should use the Oxford comma](https://developers.securedrop.org/en/latest/documentation_guidelines.html#usage-and-style), so I believe we need an additional comma after ``sudo paxtext blackhat`` (before the "and verify" part)